app/controllers/application_controller.rb

```diff
@@ -1,160 +1,143 @@
+# Superclass for all model controllers
 class ApplicationController < ActionController::Base
   protect_from_forgery
 
   helper :html
   helper :my_content
-  
+
   helper_method :nl2br
-  
+
   helper_method :universe_filter
 
-  # Rails changed cookie format in rails 4, so log out all old users so they get the new version
+  # Rails changed cookie format in rails 4, so log out all old users so that
+  # they get the new version
   rescue_from JSON::ParserError, with: :force_user_logout
   def force_user_logout
     reset_session
     redirect_to login_path
   end
-  
+
   # View Helpers
   def nl2br(string)
-    #simple_format string
-    string.gsub("\n\r","<br>").gsub("\r", "").gsub("\n", "<br />").html_safe
+    # simple_format string
+    string.gsub('\n\r', '<br>').gsub('\r', '').gsub('\n', '<br />').html_safe
   end
-  
+
   def universe_filter
-  	universes = Universe.where(user_id: session[:user])
-  	return if universes.length == 0
-  	
-  	unless @selected_universe_filter
-  	  @selected_universe_filter = 'All universes'
-  	end
-  
-  	html = '<span class="btn-group input-append help-inline">'
-  	html << '<button class="btn dropdown-toggle" data-toggle="dropdown">'
-  	html << '<i class="icon-globe"></i> ' + @selected_universe_filter + ' '
-  	html << '<span class="caret"></span>'
-  	html << '</button>'
-  	html << '<ul class="dropdown-menu dropdown-picker">'
-		html << '<li><a href="/plan/characters">All universes</a></li>'
-  	universes.each do |i|
-  		html << '<li><a href="/plan/characters/from/' + i.name.strip + '">' + i.name + '</a></li>'
-  	end
-  	html << '</ul>'
-  	html << '</span>'
-
-  	return html.html_safe
-  end
-  
+    return if Universe.where(user_id: session[:user]).length.zero?
```

Comment:
This is fine for the pull request (excellent), but just making a note we probably want scopes on `User` for each content type eventually, so we can do things like `return if User.find(session[:user]).universes.length.zero?`. (mostly a note to myself so I can jump back into code after this PR :) )